src/_shared/components/FloatingActionButton/FloatingActionButton.styles.tsx

```diff
@@ -0,0 +1,17 @@
+import { createStyles, makeStyles } from '@material-ui/core/styles';
+
+/**
+ * Styles for Floating Action Button component.
+ */
+export const useStyles = makeStyles(() =>
+  createStyles({
+    bottomRightFloating: {
+      margin: 0,
+      top: 'auto',
+      right: 50,
+      bottom: 20,
```

Comment:
Nitpick: could these be in `rem` units rather than pixels? 1 rem = 16 px in MUI 4